docs/3_software.md

```diff
@@ -4,7 +4,7 @@ Now you can write some software to verify that the circuit works. The software w
 
 ### Setting up Mbed CLI
 
-Part of the ARM Mbed IoT Device Platform is Mbed CLI, an offline compilation tool, which you will use to write and compile software. To get started:
+Part of the Pelion IoT Platform is Mbed CLI, an offline compilation tool, which you will use to write and compile software. To get started:
```

Comment:
This is not correct. Should remain 'Arm Mbed'.

Reply:
My naming standards doc says Pelion IoT Platform, but I'm not sure if that applies in this specific case since CLI is an OS thing.